web/client/api/catalog/CSW.js

```diff
@@ -210,22 +232,29 @@ export const getCatalogRecords = (records, options, locales) => {
             const ogcReferences = layerType && layerType !== 'esri'
                 ? parsedReferences[layerType]
                 : undefined;
-            return {
-                serviceType: 'csw',
-                layerType,
-                isValid: !!layerType,
-                boundingBox: record.boundingBox,
-                description: dc && isString(dc.abstract) && dc.abstract || '',
-                layerOptions: options && options.layerOptions || {},
-                identifier: dc && isString(dc.identifier) && dc.identifier || '',
-                references: references,
-                thumbnail: getThumbnailFromDc(dc, options),
-                title: dc && isString(dc.title) && dc.title || '',
-                tags: dc && dc.tags || '',
-                metadata,
-                capabilities: record.capabilities,
-                ogcReferences
-            };
+            let catRecord;
+            if (dc && dc.format === "3D Tiles") {
```

Comment:
please check if there is a constant **3D_TILES**  somewhere for this value  "3D Tiles"

if not we can add it here web\client\api\ThreeDTiles.js